.github/workflows/test.yml

```diff
@@ -0,0 +1,15 @@
+name: PHPUnit Tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  schedule:
+    - cron: '0 0 * * 1'
+
+jobs:
+  test:
+    uses: dvsa/.github/.github/workflows/php-tests.yml@v3.2.0
```

Comment:
If you bump these to use `v.3.2.2` the deprecation warnings on the workflow run will be removed